stdlib/ballerina-http/src/main/ballerina/http/http_caching_client.bal

```diff
@@ -137,8 +137,7 @@ public type HttpCachingClient object {
         P{{httpMethod}} HTTP method to be used for the request
         P{{path}} Resource path
         P{{request}} An HTTP request
-        R{{}} The inbound response message
-        R{{}} Error occurred during HTTP client invocation
+        R{{}} The response for the request or an `error` if failed to establish communication with the upstream server
     }
     public function execute(string httpMethod, string path, Request request) returns Response|HttpConnectorError;
```

Comment:
Shouldn't we change HttpConnectorError->error in all places?

Reply:
Changing HttpConnectorError to error work is still in progress